### ISSUE TITLE
Increase contrast of cyclepaths in cycle mode

### ISSUE THREAD

**Issue by matthijskooijman** [osmandapp/OsmAnd#18051]
# 🚀 feature request

### Description

In the OsmAnd rendering style, cyclepaths are currently rendered in a light blue, that is not super easy to see when zoomed out. This is probably ok for most modes, but when in cycle mode you would want to see cyclepaths more clearly, so you can quickly see what a good route would be even when zoomed out a little.

### Describe the solution you'd like
For example, using the dark blue color used by Topo, or the same color, thicker line used by the [CyclOSM layer](https://www.openstreetmap.org/#map=15/50.6347/6.0284&layers=Y):

![image](https://github.com/osmandapp/OsmAnd/assets/194491/2f00d1d3-d96a-43ea-937e-efeaa9adb12d)

**Comment by mikehgentry** [comment]
You can configure the vector maps to look a fair bit like CyclOSM

https://github.com/Louis-LJ/OsmAnd_Cycling_Map

which isn't to say there isn't room for improvement in the default configuration

**Comment by matthijskooijman** [comment]
Yeah, I'm aware of those alternative rendering styles, but they have their own issues and it would IMHO be good to improve the default rendering style in this area as well, since a lot of people will not be aware of the option of third-party rendering styles...

**Comment by DmitryAlexei** [comment]
OsmAnd~ 4.6.0#698m, released: 2023-08-30
- 50.7968830, 6.4634477
- bicycle profile

Actual result:
- the display of cycle paths starts at zoom level 14
- the thickness of the cycle paths feels almost the same for all zoom levels

<table style="width:100%">
  <tr>
<th><img src="https://github.com/osmandapp/OsmAnd/assets/116079541/3af06cab-478e-40aa-be47-2ed07ea76c8d"  width="300" height="auto"/> </th>
 <th><img src="https://github.com/osmandapp/OsmAnd/assets/116079541/3c68616a-bf0b-4eb1-b442-8eebe45d9aea"  width="300" height="auto"/> </td>
  </tr>
</table>

<table style="width:100%">
  <tr>
<th><img src="https://github.com/osmandapp/OsmAnd/assets/116079541/fd42176b-632d-42f5-bc9a-6ec794e6ff0e"  width="300" height="auto"/> </th>
 <th><img src="https://github.com/osmandapp/OsmAnd/assets/116079541/972289d4-330d-411e-a797-534c1e08645f"  width="300" height="auto"/> </th>
 <th><img src="https://github.com/osmandapp/OsmAnd/assets/116079541/afad729b-2142-4275-b95c-b40591ba0c5e"  width="300" height="auto"/> </td>
  </tr>
</table>
